content/metadata.yaml

```diff
@@ -13,12 +13,15 @@ author_info:
     orcid: XXXX-XXXX-XXXX-XXXX
     twitter: johndoe
     email: john.doe@something.com
-    affiliations: Department of Something, University of Whatever
+    affiliations:
+      - Department of Something, University of Whatever
     funders: Grant XXXXXXXX
   -
     github: janeroe
     name: Jane Roe
     initials: JR
     orcid: XXXX-XXXX-XXXX-XXXX
     email: jane.roe@whatever.edu
-    affiliations: Department of Whatever, University of Something
+    affiliations:
+      - Department of Something, University of Whatever
```

Comment:
Great, I was going to suggest you make this change and see that you already did.